packages/sitebuilder/src/wizards/look-and-feel/screens/ColorSelection.tsx

```diff
@@ -113,7 +113,6 @@ const ColorSelection = () => {
 										))
 									}
 								</RadioGroup>
-								<ModalDeviceSelection />
```

Comment:
@ColtenVanTussenbrook I removed the device selection component from here because I didn't see it in the designs and it wasn't in the previous version of this feature.

Reply:
Good call!